snackbars/src/main/res/values/styles-snackbar.xml

```diff
@@ -14,5 +14,7 @@
     </style>
 
     <style name="Widget.Vitamin.SnackbarButton" parent="Widget.Vitamin.Button.GhostReversed.Medium">
+        <item name="android:layout_marginTop">@dimen/vtmn_snackbars_padding_vertical</item>
```

Comment:
You used `@dimen/vtmn_snackbars_padding_vertical` which is 8dp.
But in Figma, padding depends the text. @SimonLeclercq @Sabrinavigil @MARIEDELATTRE can you validate if it is OK to set 8px for the Button vertical padding ? 
<img width="629" alt="image" src="https://user-images.githubusercontent.com/46710837/231190623-01c982a5-696c-46b4-9a8f-f3fae11b92de.png">
<img width="930" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/46710837/231192358-a5055368-7cdd-4e91-991d-dff1d860b292.png">
<img width="881" alt="Pasted Graphic 2" src="https://user-images.githubusercontent.com/46710837/231192427-963d63af-f49d-4b3c-be07-eb0b192bec19.png">

Reply:
Based on the material documentation, the padding should be 6px -> https://m2.material.io/components/snackbars#specs